Sources/BraintreeVisaCheckout/BTVisaCheckoutAnalytics.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+enum BTVisaCheckoutAnalytics {
```

Comment:
This is a separate ticket. Anything related to analytics can be removed from this PR.